assets/js/modules/ads/components/common/AdsConversionIDTextField.js

```diff
@@ -66,8 +66,15 @@ export default function AdsConversionIDTextField() {
 			<h4 className="googlesitekit-settings-module__fields-group-title">
 				{ __( 'Conversion ID', 'google-site-kit' ) }
 			</h4>
+
+			{ helperText && (
+				<p className="googlesitekit-settings-module__fields-group-helper-text">
+					{ helperText }
+				</p>
+			) }
+
 			<TextField
-				label={ __( 'Ads Conversion ID', 'google-site-kit' ) }
+				label={ __( 'Conversion ID', 'google-site-kit' ) }
```

Comment:
Hey @techanvil, the copy has changed in Figma since I worked on this ticket. I know there was a conversation in a stand-up with Evan about changing this so it's now filtered through to the designs. I'll review and update all copy to the latest in Figa.